zhaquirks/tuya/mcu/__init__.py

```diff
@@ -161,13 +161,15 @@ class TuyaConnectionStatus(t.Struct):
         tsn: t.uint8_t
         status: t.LVBytes
 
-    attributes = TuyaNewManufCluster.attributes.copy()
-    attributes.update(
-        {
-            # MCU version
-            ATTR_MCU_VERSION: ("mcu_version", t.uint48_t, True),
-        }
-    )
+    class AttributeDefs(TuyaNewManufCluster.AttributeDefs):
+        """Attribute Definitions."""
+
+        mcu_version = foundation.ZCLAttributeDef(
+            id=ATTR_MCU_VERSION,
+            type=t.uint48_t,
+            access=foundation.ZCLAttributeAccess.Read,
+            is_manufacturer_specific=True,
+        )
```

Comment:
I completely missed that there were some `attributes` in the `TuyaMCUCluster` before.
Can we move this up to the top of the cluster class, like it is in other custom clusters?